generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -96,7 +96,18 @@ quarkus.oidc.authentication.cookie-path=/
 quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
 quarkus.oidc.authentication.restore-path-after-redirect=false
 
+
+jhipster.oidc.logout-url=http://localhost:9080/auth/realms/jhipster/protocol/openid-connect/logout
+
+quarkus.http.auth.permission.public.paths=/management/prometheus, /management/prometheus/*, /management/health, /management/info, /
+quarkus.http.auth.permission.public.policy=permit
+
+quarkus.http.auth.permission.authenticated.paths=/**, /oauth2/authorization/oidc
```

Comment:
I assume `/**` also restricts the UI paths. Since all backend resources have path starting `/api`, can we restrict `/api/**`. I am not sure if there is a spring boot actuator equivalent path like `/management/**`. If yes, we can also restrict that.